test/test_images/performance/main.go

```diff
@@ -44,10 +47,20 @@ var (
 	warmupSeconds uint
 
 	// role=aggregator
-	expectRecords uint
-	listenAddr    string
+	expectRecords  uint
+	listenAddr     string
+	additionalTags Tags
 )
 
+func (tags *Tags) String() string {
+	return fmt.Sprint(*tags)
+}
+
+func (tags *Tags) Set(value string) error {
+	*tags = append(*tags, value)
```

Comment:
You need to split  the string and convert it to your type before doing that assignment.

Reply:
I'd day the usage of set is to set a single value not a list of values